### DISCUSSION TITLE
EC encoded public key to EVP_PKEY

### DISCUSSION THREAD

**Discussion by dnssoftware** [openssl/openssl#23994] [Q&A] [answered]
I’ve got an EC public key in encoded format - x04 + 64 bytes, 65 in total.

I know how to create an EVP_PKEY using legacy APIs and use it successfully to do signature verifications etc.

I am struggling to get EVP_PKEY by using non deprecated APIs.

Any pointers please?

**Comment by t8m** [comment, marked as answer]
Please see this discussion: https://github.com/openssl/openssl/discussions/23820
